agent/main.go

```diff
@@ -164,7 +164,7 @@ func main() {
 		}()
 	}
 
-	ticker := time.NewTicker(time.Duration(opts.KeepAliveInterval) * time.Second)
+	ticker := time.NewTicker(time.Duration(10 * time.Second))
```

Comment:
Is this intended?

Reply:
WIP

Reply:
I'll keep this open so we don't forget to solve this prior to merging it.